libs/dh/globalization/configuration-localization/src/lib/dh-transloco-testing.module.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license
+ * Copyright 2021 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  TranslocoTestingModule,
+  TranslocoTestingOptions,
+} from '@ngneat/transloco';
+import {
+  da,
+  en,
+} from '@energinet-datahub/dh/globalization/assets-localization';
+
+export function getTranslocoTestingModule(
```

Comment:
suggestion: Delete this file and the associated export. The functions is not used.